pennylane/templates/subroutines/phase_adder.py

```diff
@@ -37,28 +37,30 @@ class PhaseAdder(Operation):
 
     .. math::
 
-         \text{PhaseAdder}(k,mod) |\phi (x) \rangle = |\phi (x+k \; \text{modulo} \; mod) \rangle,
+        \text{PhaseAdder}(k,mod) |\phi (x) \rangle = |\phi (x+k \; \text{mod} \; mod) \rangle,
 
-    where :math:`|\phi (x) \rangle` represents the :math:`| x \rangle` : state in the Fourier basis,
+    where :math:`|\phi (x) \rangle` represents the :math:`| x \rangle` state in the Fourier basis,
 
-     .. math::
+    .. math::
 
-         \text{QFT} |x \rangle = |\phi (x) \rangle.
+        \text{QFT} |x \rangle = |\phi (x) \rangle.
 
     The implementation is based on the quantum Fourier transform method presented in
     `arXiv:2311.08555 <https://arxiv.org/abs/2311.08555>`_.
 
     .. note::
 
-        Note that :math:`x` must be smaller than :math:`mod` to get the correct result. Also, when
-        :math:`mod \neq 2^{\text{len(x\_wires)}}` we need :math:`x < 2^{\text{len(x\_wires)}}/2`,
-        which means that we need one extra wire in ``x_wires``.
+        To obtain the correct result, :math:`x` must be smaller than :math:`mod`. Also, when
+        :math:`mod \neq 2^{\text{len(x_wires)}}` it is needed :math:`x < 2^{\text{len(x_wires)}-1}`,
+        which means that one extra wire in ``x_wires`` is required.
+
+    .. seealso:: :class:`~.QFT` and :class:`~.Adder`.
 
     Args:
         k (int): the number that needs to be added
         x_wires (Sequence[int]): the wires the operation acts on
-        mod (int): the modulus for performing the addition, default value is :math:`2^{len(x\_wires)}`
-        work_wire (Sequence[int]): the auxiliary wire to be used for performing the addition
+        mod (int): the modulo for performing the addition. If not provided, it will be set to :math:`2^{\text{len(output_wires)}}`.
```

Comment:
```suggestion
        mod (int): the modulo for performing the addition. If not provided, it will be set to :math:`2^{\text{len(output_wires)}}`
```